app/pages/lab/visibility.cjsx

```diff
@@ -169,6 +169,9 @@ module.exports = React.createClass
             {@state.workflows.map (workflow) =>
               setting = workflow.active
               stats_completeness_type = workflow.configuration.stats_completeness_type ? 'retirement'
+              stats_visible = !workflow.configuration.stats_hidden
+              if !workflow.active
+                stats_visible = !(workflow.configuration.stats_hidden == undefined) && stats_visible
```

Comment:
There is a `!` in front of the first condition, so it becomes `false && true` and sets `stats_visible` to `false` (now that I think of it I should have used `!=`, my brain is being slow today).  But you make a good point that using `workflow.active` as the initial value is less confusing.